docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -12,21 +12,52 @@ keywords:
 
 # Documentation structure
 
-The technical writing team makes very intentional decisions about how we organize and structure product documentation. Users can find what they need quickly and easily with well-structured content.
+The Grafana Labs Docs team makes intentional decisions about how to organize and structure product documentation. With well-structured content, users can find what they need quickly and easily.
 
-Information architecture (IA) refers to the way in which content is organized. Before you start contributing to product documentation, it is important to understand the IA of the content.
+Before you begin contributing to documentation, it is important to understand the structure of the content.
 
-Generally, the IA determines how content is:
+According to STC’s [Information Design and Information Architecture: Why Technical Communicators Should Care About These Fields](https://www.stc.org/intercom/2022/05/information-design-and-information-architecture-why-technical-communicators-should-care-about-these-fields/),
+information architecture is "...the practice of organizing, structuring, and labeling information to it’s easy to find, use, and understand..."
+
+Generally, documentation structure determines how content is:
```

Comment:
Not coming up with an alternative to this incomplete phrase that stands alone. How can line 22 be made complete on its own?

Reply:
Line 22 is the introduction to the three bullets that follow underneath it. Those bullets incorporated into a complete sentence. I can add punctuation to make that clearer, if you like.

Reply:
Yes, that is clear. It is a small anti-pattern wrt style that I will leave as-is for this PR.

Reply:
Also, it has a nice zen ring to it as is. :)